shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm

```diff
@@ -224,7 +224,11 @@ - (NSDictionary*)getClipboardData:(NSString*)format {
 
 - (void)setClipboardData:(NSDictionary*)data {
   UIPasteboard* pasteboard = [UIPasteboard generalPasteboard];
-  pasteboard.string = data[@"text"];
+  if (data[@"text"]) {
+    pasteboard.string = data[@"text"];
+  } else {
+    pasteboard.string = @"null";
```

Comment:
Passing an empty string may be better than "null" here. If the contents were actually the string "null" we would not be able to distinguish between an incorrect usage of the setClipboardData API vs the actual content. End users would be very surprised to see the term "null" if this was accidentally used improperly.